compiler/dalgona/CMakeLists.txt

```
@@ -24,12 +24,12 @@ else()
   endif()
 
   # Require same python version of common-artifacts
-  if(Python_VERSION VERSION_GREATER_EQUAL 3.12)
-    message(STATUS "Build dalgona: FALSE (Python version 3.12 or higher is not supported yet)")
+  if(Python_VERSION VERSION_GREATER_EQUAL 3.13)
+    message(STATUS "Build dalgona: FALSE (Python version 3.13 or higher is not supported yet)")
```

Do you have any specific reason to use the work "FALSE" instead if "FAILED"?

```suggestion
    message(STATUS "Build dalgona: FAILED (Python version 3.13 or higher is not supported yet)")
```

---

1/ it is as-is, not from me :)
2/ my guess; build didn't fail by some problem but build is disabled in purpose